app/src/main/res/layout/fragment_calendar.xml

```diff
@@ -1,12 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout
+<LinearLayout
```

Comment:
We generally make a new story for a different variant of a component to make it easy for the developers to refer to the usage. Please see `ComponentRegistry.kt` for an example

Reply:
Cool, I will move it to a new story.